app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
The prefix was from the style guide https://angular.io/styleguide#!#02-07
Quote:
- Why? Prevents element name collisions with components in other apps and with native HTML elements.
- Why? Makes it easier to promote and share the component in other apps.
- Why? Components are easy to identify in the DOM.

We have discussed this at length already and I'm prepared to change my stance on this. I don't really see using modular design being needed only when a component is already needed in other places. Especially if it's this easy to achieve beforehand. Seems like our design principles are just a bit different on this?